packages/personal-data-erasure/src/main.js

```diff
@@ -203,7 +199,6 @@ export default class PersonalDataErasure {
     const resourcesToDelete = [
       customersResource,
       ordersResource,
-      cartsResource,
```

Comment:
This one would also cause errors since it gets deleted inside the `customerResource` already. Sometimes it would be fast enough to delete it before deletion of `customer` was finished, sometimes not.